doc/rados/operations/cache-tiering.rst

```diff
@@ -64,6 +64,23 @@ configure how this migration takes place. There are two main scenarios:
   might contain out-of-date data provides weak consistency. Do not use 
   ``readonly`` mode for mutable data.
 
+And the modes above are accomodated to adapt different configurations:
+
+- **Read-forward Mode:** this mode is the same as the ``writeback`` mode
+  when serving write requests. But when Ceph clients is trying to read objects
+  not yet copied to the cache tier, Ceph **forward** them to the backing tier by
+  replying with a "redirect" message. And the clients will instead turn to the
+  backing tier for the data. If the read performance of the backing tier is on
+  a par with that of its cache tier, while its write performance or endurance
+  falls far behind, this mode might be a better choice.
+
```

Comment:
Maybe clients will will have a stale read here? e.g. the newly writes are not flushed to the back tier from cache tier, but the read requests are serviced from base tier.